modules/coin/rest/handlers.go

```diff
@@ -118,6 +121,25 @@ func doSend(w http.ResponseWriter, r *http.Request) {
 	common.WriteSuccess(w, tx)
 }
 
+type RoleInput struct {
```

Comment:
Thanks for pointing that out. So this PR is based off #208 and since it was taking long to get it reviewed I based this branch of prior versions of it and made this one, with RoleInput still in modules/coin/rest instead of modules/roles/rest. Once #208 is merged, I'll rebase it here and it'll be gone :)